### PR TITLE
Migration notebook updated

### PR DESCRIPTION
**Pull request by willrayeo**
- The cells and markdown associated with generating an access token (replacing keycloak token) have been updated.
- The method to download a tile using OData has also been updated to be less memory intensive.

### CODE REVIEW

File: sentinelhub/migration_from_scihub_guide.ipynb [hunk 1]
```diff
@@ -310,51 +319,62 @@
...
+    "### Retrieving your Access Token"
    ]
   },
   {
    "cell_type": "markdown",
    "id": "17b2e052-6962-4881-8fd0-de9ad3dc5771",
    "metadata": {},
    "source": [
-    "Whilst no credentials are needed to search the catalogue; in order to download products from Copernicus Data Space Ecosystem catalogue using OData and OpenSearch API users are required to have a Keycloak token. This token can be generated in both Linux and Window OS using either `cURL` or `python` script. You can generate this token using the following code block (more information can be found in our [documentation](https://documentation.dataspace.copernicus.eu/APIs/Token.html)).\n",
+    "Whilst no credentials are needed to search the catalogue; in order to download products from Copernicus Data Space Ecosystem catalogue using OData and OpenSearch API users are required to have a Access token. This token can be generated in both Linux and Window OS using either `cURL` or `python` script. You can generate this token using the following code block (more information can be found in our [documentation](https://documentation.dataspace.copernicus.eu/APIs/Token.html)).\n",
```
Comment (Kampen1988) [inline sentinelhub/migration_from_scihub_guide.ipynb:330]: Not a native speaker, but I'd put "an Access token" here.

File: sentinelhub/migration_from_scihub_guide.ipynb [hunk 2]
```diff
@@ -928,9 +948,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Sentinel Hub",
+   "display_name": "Python 3 (ipykernel)",
    "language": "python",
-   "name": "sentinelhub"
+   "name": "python3"
```
Comment (Kampen1988) [inline sentinelhub/migration_from_scihub_guide.ipynb:953]: Please correct the kernelspec metadata according to the [instructions in the repo](https://github.com/eu-cdse/notebook-samples#kernel-spec-metadata).

**Review by Kampen1988 — CHANGES_REQUESTED** [review]

File: sentinelhub/migration_from_scihub_guide.ipynb (hunk 1 above)
Comment (willrayeo) [reply sentinelhub/migration_from_scihub_guide.ipynb:330]: updated "a" to "an"

File: sentinelhub/migration_from_scihub_guide.ipynb (hunk 2 above)
Comment (willrayeo) [reply sentinelhub/migration_from_scihub_guide.ipynb]: resolved.

**Pull request merged**